pkg/apis/core/types.go

```diff
@@ -3461,6 +3461,64 @@ type PodTemplateList struct {
 	Items []PodTemplate
 }
 
+// BackoffPolicyAction specifies how a Pod failure is handled.
+type BackoffPolicyAction string
+
+const (
+
+	// This is an action which might be taken on pod failure - mark the
+	// pod's job as Failed and terminate all running pods.
+	BackoffActionTerminate BackoffPolicyAction = "Terminate"
+
+	// This is an action which might be taken on pod failure - the pod will be
+	// restarted and the counter for .backoffLimit will not be incremented.
+	BackoffActionIgnore BackoffPolicyAction = "Ignore"
+)
+
+type BackoffPolicyOnExitCodesOperator string
+
+const (
+	BackoffPolicyOnExitCodesOpIn    BackoffPolicyOnExitCodesOperator = "In"
+	BackoffPolicyOnExitCodesOpNotIn BackoffPolicyOnExitCodesOperator = "NotIn"
+)
+
+type BackoffPolicyOnExitCodesRequirement struct {
+	// Restricts the check for exit codes to only apply to the container with
+	// the specified name. When empty the rule applies to all containers
+	// +optional
+	ContainerName *string
+
+	// Represents the relationship between the container exit code(s) and the
+	// specified values.
```

Comment:
Constants are not part of the generated documentation for the website. Then, you have to repeat here what are the possible operators and their semantics.

Reply:
done